docs/docs/monitoring/analytics.mdx

```diff
@@ -72,17 +72,57 @@ will be disabled.
 
 ### Connect a data warehouse
 
+You can choose to configure the analytics pipeline to stream the transformed conversational data to two different data warehouse types in AWS:
+
+- [PostgreSQL](#postgresql)
+- [Redshift](#redshift)
+
+
 #### PostgreSQL
 
-TODO: general instructions.
+You can use Amazon Relational Database Service (RDS) to create a PostgreSQL DB instance which is the environment that will run your PostgreSQL database.
+
+First you must set up Amazon RDS by completing the instructions listed [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_SettingUp.html).
+Next, create the PostgreSQL DB instance. You can follow one of the following instruction sets:
+
+- the AWS *Easy create* instructions listed in the [**Creating a PostgreSQL DB instance** section](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html#CHAP_GettingStarted.Creating.PostgreSQL)
+- the AWS [*Standard create* instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html)
 
-TODO: instructions for AWS RDS
+In the process, you must decide on a database authentication option from the below:
+
+- **Password authentication** to use database credentials only, in which case you must enter a username for the master username, as well as generate the master password.
+- [**Password and IAM DB authentication**](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html) to use IAM users and roles for authentication fo database users.
+- [**Password and Kerberos authentication**](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/postgresql-kerberos.html)
+
+You will need these database credentials to build the DB URL in the DBAPI format specified in the [**Connect an assistance** section](#connect-an-assistant).
```

Comment:
let's rephrase to something that ask users to create a user/password for use in the DB URL